### ISSUE TITLE
Containerd package shouldn't provide runc which is packaged by distros already

### ISSUE THREAD

**Issue by Romain-Geissler-1A** [docker/packaging#101]
Hi,

As a continuation of the issue in the current repository https://github.com/docker/containerd-packaging/issues/210 I think we should simply stop trying to compile/package runc here. We are in 2023, distros now all have a runc package I guess, which hopefully is well maintained. Installing containerd with runc already installed shouldn't result in the system runc package to be uninstalled (making typically a co-installation with podman fail).

Cheers,
Romain

**Comment by crazy-max** [comment]
Agree. I have a started working on it in https://github.com/docker/packaging/compare/main...crazy-max:docker-packaging:runc-package while making a [package for BuildKit](https://github.com/docker/packaging/tree/main/pkg/buildkit) because today it also depends on runc.

**Comment by Romain-Geissler-1A** [comment]
From what I see, you split runc in a separate package, but still you package it yourself. Shouldn't this task be left to the distros instead (who package it already) ?

**Comment by crazy-max** [comment]
Well I think it's better to have a separate package and make it available on download.docker.com. OS one if often outdated.

**Comment by Romain-Geissler-1A** [comment]
Mmmh honestly I think this is some work you will try to maintain on your side which in quite some cases won't even be used. By experience, what you want to do works with .deb, but not with .rpm. For RHEL (the distro I care about), runc uses epoch 4 (see https://gitlab.com/redhat/centos-stream/rpms/runc/-/blob/e1b1ec00e2a4f8f7cb1589ea65207ba0fa8e8864/runc.spec#L20) so no matter how recent is the version you will package, the official RHEL package will always "win" against yours, as it will have a bigger epoch (you specify no epoch in your runc.spec file).

**Comment by crazy-max** [comment]
Yes indeed, the branch on docker/packaging is still a draft and needs more work to take into account epoch and also changes for other packages to make them dependent on it.
